scripts/distributed/tle_on.sh

```diff
@@ -69,8 +69,8 @@ done
 shift "$((OPTIND - 1))"
 
 if [[ "$enable_tle" == "yes" ]]; then
-	export GLIBC_ELISION_ENABLE=yes
+	export GLIBC_TUNABLES=glibc.elision.enable=1
```

Comment:
Do you believe that there could be users of the advance-toolchain that avoid the use of this script (`scripts/distributed/tle_on.sh`), and, instead, manually set `GLIBC_ELISION_ENABLE` on their environments?  Such users would be affected by this change, because their environment variables would cease to produce the behaviour they expect.

Is this a problem?

Reply:
@gftg85 I don't think the behavior you're describing is a problem.

However, I'm afraid this script won't work anymore on AT <= 11.0, because they still use `GLIBC_ELISION_ENABLE`.

Reply:
@tuliom this commit also changes the patch used to enable TLE on AT <= 11.0 in order to work with this script.

Reply:
>Our documentation only explains how to use TLE using our script

OK, then.